class/resource-locker.yml

```diff
@@ -1,10 +1,11 @@
 parameters:
   kapitan:
     dependencies:
-      - type: https
-        source: 'https://redhat-cop.github.io/resource-locker-operator/resource-locker-operator/resource-locker-operator-${resource_locker:charts:resource-locker-operator}.tgz'
-        unpack: true
-        output_path: dependencies/resource-locker/helmcharts
+      - type: helm
+        source: https://redhat-cop.github.io/resource-locker-operator
+        chart_name: resource-locker-operator
+        version: ${resource_locker:charts:resource-locker-operator}
+        output_path: dependencies/resource-locker/helmcharts/resource-locker-operator
```

Comment:
Due to recent changes in Commodore, downloaded artifacts must be place on versioned locations. Otherwise we will see unexpected results.

See https://syn.tools/syn/references/style-guide.html#_defining_dependencies.

Reply:
This doesn't apply to `helm` dependencies, since there changed verions are correctly picked up.